### ISSUE TITLE
javascript meetup exercise test case

### ISSUE THREAD

**Issue by finalight** [exercism/problem-specifications#1838]
one of the test case seems to be incorrect

```
  test('monteenth of August 2013', () => {
    expect(meetup(2013, 8, 'teenth', 'Monday')).toEqual(new Date(2013, 7, 19));
  });
```

if i follow the readme, the correct test case would be

```
  test('monteenth of August 2013', () => {
    expect(meetup(2013, 8, 'teenth', 'Monday')).toEqual(new Date(2013, 7, 11));
  });
```

i assume it would be the second monday of the month.
if not, can someone please explain to me in layman terms ? I don't quite understand about 'monteenth, tuesteenth, wednesteenth, thursteenth, friteenth, saturteenth, sunteenth'. I know they are madeup words, but how can i represent in terms of the date selection?

**Comment by glennj** [comment]
The Monday (or whichever day) between the 13th and the 19th (inclusive) of the month.

**Comment by petertseng** [comment]
Since the test case is correct as-is, and an explanation has been provided, which are the two things at hand in this issue, everything in this issue has been addressed and I would close it.
